### DISCUSSION TITLE
Upload stuck at “Posting, please wait…”

### DISCUSSION THREAD

**Discussion by SievaFliesAlone** [pixelfed/pixelfed-rn#94] [Bug Reports]
Upload is stuck after trying to create a new post with a large pucture.

* in Gallery, choose a picture (8.8 MB), share via Pixelfed
* add description and hashtags
* press Post

Actual result:
![Screenshot_20240721-075159](https://github.com/user-attachments/assets/06ed3a5e-4852-44ea-b053-f3425649af1c)

I've had this issue with the old app and the new app, have to use PixelDroid for uploads as a workaround.

Android 14, Pixel 6, GrapheneOS

**Comment by docpop** [comment]
I had this happen with my iphone XR (ios 18.0). After waiting about 4 minutes, I force-quit the app and came back in. The post was gone and so was all the alt-text I had written. It would be nice if there was a failure mode that could detect when something hasn't been uploaded and saves it as a draft.

**Comment by bucknam** [comment]
This has also happened to me, running iOS 18.1 on an iPhone 13 Pro.
